actions/info.action.ts

```diff
@@ -77,7 +86,7 @@ export class InfoAction extends AbstractAction {
     );
   }
 
-  async readProjectPackageJsonDependencies(): Promise<PackageJsonDependencies> {
+  async readProjectLockDependencies(): Promise<PackageJsonDependencies> {
```

Comment:
Insignificant, but I think this would be more clear as `readProjectLockfileDependencies`.

Reply:
Probably, but I was also trying to keep the name from getting too long.

Reply:
I agree, perhaps it's worth including this as a comment? Just because, a "project lock" doesn't exactly refer to anything at all.

Reply:
I think the fact that 3 lines later you see `package-lock.json` or `yarn.lock` makes the intention clear. Though I guess in a stack trace it could be _somewhat_ problematic, though a comment wouldn't help that either

Reply:
What if you removed the word `project`? It's probably the first assumption once you see "lockfile," although I suppose it doesn't _have_ to be a project.